### ISSUE TITLE
Getting R Plots Working

### ISSUE THREAD

**Issue by RehanSaeed** [dotnet/BenchmarkDotNet#567]
I've followed [this](http://benchmarkdotnet.org/Configs/Exporters.htm) guide to get the plot images using R but I can't get them to appear. These are the steps I followed:

1. Add `[CsvMeasurementsExporter]` and `[RPlotExporter]` attributes to my benchmark.
2. Add an `R_HOME`environment variable pointing at my Visual Studio installation of R in `C:\Program Files\Microsoft\R Client\R_SERVER`.
3. Runing my three benchmarks causes one `BuildPlots.R` file to be output but no images appear.

What am I missing? My Benchmark can be found [here](https://github.com/ASP-NET-Core-Boilerplate/Framework/tree/master/Benchmarks/Boilerplate.Mapping.Benchmark).

**Comment by adamsitnik** [comment]
Hello @RehanSaeed 

I took a look at your benchmarks, you need to use `[CsvMeasurementsExporter]` instead of `[CsvExporter]`. The former contains more information, which is mandatory to print the plots.

If this does not help please feel free to reopen the issue.

**Comment by alexandretperez** [comment]
Not working for me.

Here is the config:
```C#
    public class TestConfig : ManualConfig
    {
        public TestConfig()
        {
            Add(RankColumn.Roman);
            Add(CsvMeasurementsExporter.Default);
            Add(RPlotExporter.Default);

            Set(new DefaultOrderProvider(SummaryOrderPolicy.FastestToSlowest));
        }
    }

// then on the class

        [Config(typeof(TestConfig))]
        public class Md5VsSha256
        {
            private const int N = 10000;
            private readonly byte[] data;

            private readonly SHA256 sha256 = SHA256.Create();
            private readonly MD5 md5 = MD5.Create();

            public Md5VsSha256()
            {
                data = new byte[N];
                new Random(42).NextBytes(data);
            }

            [Benchmark]
            public byte[] Sha256() => sha256.ComputeHash(data);

            [Benchmark]
            public byte[] Md5() => md5.ComputeHash(data);
        }

```

Then the console show the message:

```
// ***** BenchmarkRunner: Finish  *****

// * Export *
  BenchmarkDotNet.Artifacts\results\BenchTest.Program+Md5VsSha256-report.csv
  BenchmarkDotNet.Artifacts\results\BenchTest.Program+Md5VsSha256-report-github.md
  BenchmarkDotNet.Artifacts\results\BenchTest.Program+Md5VsSha256-report.html
  BenchmarkDotNet.Artifacts\results\BenchTest.Program+Md5VsSha256-measurements.csv
  BuildPlots.R
  BenchmarkDotNet.Artifacts\results\Md5VsSha256-boxplot.png
  BenchmarkDotNet.Artifacts\results\Md5VsSha256-barplot.png
```

But when I go the the results directory, there's no *.png files.

**Comment by adamsitnik** [comment]
@alexandretperez could you upload your `.log` file to https://gist.github.com/ ?

**Comment by alexandretperez** [comment]
@adamsitnik
https://gist.github.com/alexandretperez/cbec92c000196f687f422eb9d22c33f0

**Comment by AndreyAkinshin** [comment]
@alexandretperez, it's strange that you don't have any errors.
Do you have installed R? Do you have `Rscript.exe` in your `%PATH%` (see also: [docs/Exporters#Plots](http://benchmarkdotnet.org/Configs/Exporters.htm#plots)) (just try to run `Rscript.exe` in command line from any folder). If it's available, try to manually run generated R script (`BuildPlots.R`) via `Rscript.exe`.

**Comment by alexandretperez** [comment]
@AndreyAkinshin, running "Rscript.exe BuildPlots.R" through the command line worked.  Thanks!

**Comment by AndreyAkinshin** [comment]
@alexandretperez, it turned out that we had a bug in RPlotExporter (see https://github.com/dotnet/BenchmarkDotNet/issues/623). I guess you had exactly this problem. It was fixed in v0.10.12.
